hw_1.2/hm_1.2/Program.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Text;
+
+namespace hm_1._2
+{
+    class Program
+    {
+        //calculate BWT
+        static string GetBWT(string str)
```

Comment:
```suggestion
        static (string encodedString, int endPosition) GetBWT(string str)
```
Зачем добавлять индекс конца строки в саму строку, если можно (и нужно) строку и индекс раздельно?